### PR TITLE
Add Kirill Podoprigora for AST codeowners

### PR DESCRIPTION
**Pull request by Eclips4**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -38,7 +38,6 @@ Python/compile.c              @markshannon @iritkatriel
 Python/assemble.c             @markshannon @iritkatriel
 Python/flowgraph.c            @markshannon @iritkatriel
 Python/instruction_sequence.c @iritkatriel
-Python/ast_opt.c              @isidentical
```
Comment (Eclips4) [inline .github/CODEOWNERS:41]: I moved it under the `ast` section.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -158,10 +157,12 @@ Include/internal/pycore_time.h  @pganssle @abalkin
 /Tools/cases_generator/        @markshannon
 
 # AST
-Python/ast.c                  @isidentical @JelleZijlstra
-Parser/asdl.py                @isidentical @JelleZijlstra
-Parser/asdl_c.py              @isidentical @JelleZijlstra
-Lib/ast.py                    @isidentical @JelleZijlstra
+Python/ast.c                  @isidentical @JelleZijlstra @eclips4
+Python/ast_opt.c              @isidentical @eclips4
```
Comment (ezio-melotti) [inline .github/CODEOWNERS:161]: For consistency should @JelleZijlstra be added here to? 

```suggestion
Python/ast_opt.c              @isidentical @JelleZijlstra @eclips4
```

**Review by ezio-melotti — APPROVED** [review]

File: .github/CODEOWNERS (hunk 2 above)
Comment (Eclips4) [reply .github/CODEOWNERS:161]: I guess when he added himself as an`ast` codeowner, he just forgot about the `ast_opt` file.
But let's wait for his opinion.

Comment (JelleZijlstra) [reply .github/CODEOWNERS:161]: I didn't forget, I'm less interested in that file.

**Review by JelleZijlstra — APPROVED** [review]
Welcome onboard!

**Pull request merged**